scapy/utils.py

```diff
@@ -1171,11 +1172,11 @@ def __call__(cls, filename):  # type: ignore
         raise Scapy_Exception("Not a supported capture file")
 
     @staticmethod
-    def open(fname  # type: Union[IO[bytes], str]
+    def open(fname  # type: Union[IO[bytes], str, pathlib.PosixPath, pathlib.WindowsPath]
```

Comment:
```suggestion
    def open(fname  # type: Union[IO[bytes], str, bytes, os.PathLike]
```
See https://peps.python.org/pep-0519/#provide-specific-type-hinting-support